CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Add nanoseconds to the log
```

Comment:
`Log messages are now showing microseconds`. (@sigalsax ?)

please also add a link to an issue (and create one if there isn't one.

Reply:
@abrahamko @orenbm `Log messages are now showing microseconds to allow for clarity and easier troubleshooting` 

IMO we should also add the motivation behind each CHANGELOG entry not just _what_ was changed